docs/src/tutorials/tutorial_6.md

```diff
@@ -0,0 +1,313 @@
+What I mean by this is: If you move an object from `A` to `B` using [`Translation`](@ref) it would do so in a linear and therefore boring fashion.
+
+We'll create an interesting loading animation for this. It consists of five circles which move from the center to the outside rotate around the center and back to the origin. During this process they appear and disappear as well as changing color.
+
+## Learning Outcomes
+
+This tutorial shows some more power of subactions that we introduced in v0.2. Combined with the power of the awesome library [Animations.jl](https://github.com/jkrumbiegel/Animations.jl) you have very fine grained control over the objects you animate and can benefit from the easiness of easing functions. 😄
+
+Today you'll learn how to
+- use easing functions to have animations with pep
+- have full control over the animation of objects
+- create a beautiful (if I can say so myself) loading animation
+
+## Moving with Different Speed
+
+The main point of this tutorial is to explain how to basically "interact" with objects with a non-linear speed. Our first step will be to rotate a circle around the origin at a increasing and then decreasing speed. 
+
+Let's start with the boring linear animation:
+
+```julia
+using Javis
+
+function ground(args...)
+    background("black")
+    sethue("white")
+end
+
+video = Video(600, 400)
+
+actions = [
+    Action(
+        (args...) -> circle(Point(150, 0), 20, :fill);
+        subactions = [
+            SubAction(Rotation(0.0, 2π)),
+        ],
+    )
+]
+
+javis(
+    video,
+    [BackgroundAction(1:200, ground), actions...],
+    pathname = "loading.gif",
+)
+```
+
+![Linear movement](assets/loading_circle_linear.gif)
+
+I would say that this looks a bit dull. Let us rotate it with varying speeds. For this I'll use the `sineio` easing function. You can try another easing function.
+They are all described [here](https://jkrumbiegel.github.io/Animations.jl/stable/#Easings-1).
+
+First of all we need `Animations` for this and I also load `Colors` here for later.
+
+```julia
+using Javis, Animations, Colors
+
+function ground(args...)
+    background("black")
+    sethue("white")
+end
+
+rotate_anim = Animation(
+    [0, 1], # must go from 0 to 1
+    [0, 2π],
+    [sineio()],
+)
+
+video = Video(600, 400)
+
+actions = [
+    Action(
+        (args...) -> circle(Point(150, 0), 20, :fill);
+        subactions = [
+            SubAction(rotate_anim, rotate()),
+        ],
+    )
+]
+
+javis(
+    video,
+    [BackgroundAction(1:200, ground), actions...],
+    pathname = "loading.gif",
+)
+```
+
+![Rotation with varying speed](assets/loading_circle_sineio.gif)
+
+I think that looks more interesting. The [Animations.jl](https://jkrumbiegel.github.io/Animations.jl/stable) package is very powerful so you might wanna head over to their documentation and try different things.
```

Comment:
I like that you explain Animations a little here but I think you should also go further into detail about what Animations and Colors are doing.

Reply:
Is this resolved?